game-core/src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java

```diff
@@ -35,7 +38,60 @@
 /**
  * Logic to fire rockets.
  */
-public class RocketsFireHelper {
+public class RocketsFireHelper implements Serializable {
+  private static final long serialVersionUID = -3694482580987118444L;
+  private final Set<Territory> attackingFromTerritories = new HashSet<>();
+  private final Map<Territory, Territory> attackedTerritories = new LinkedHashMap<>();
+  private final Map<Territory, Unit> attackedUnits = new LinkedHashMap<>();
+  private boolean needToFindRocketTargets;
+
+  private RocketsFireHelper() {}
+
+  /**
+   * Set up rockets for better than V1 rockets.
+   */
+  public static RocketsFireHelper setUpRockets(final IDelegateBridge bridge) {
+    final GameData data = bridge.getData();
+    final RocketsFireHelper object = new RocketsFireHelper();
```

Comment:
Probably could use a better variable name than object.

Reply:
In theory. Given that we've hidden the constructor and can only get the object by calling this static method I thought it wasn't that unreasonable.

Can you suggest something?